package.json

```diff
@@ -56,6 +57,7 @@
     "tailwind-merge": "2.5.5",
     "tailwindcss-animate": "1.0.7",
     "use-dehydrated-state": "0.1.0",
+    "usehooks-ts": "3.1.0",
```

Comment:
now that this lib is added can you pls update `use-long-timeout.ts` to use `useIsomorphicLayoutEffect` from there instead of defining its own